gui/src/components/preboarding/login.vue

```diff
@@ -168,13 +168,16 @@ export default class CortxLogin extends Vue {
             res.authorization
           );
           localStorage.setItem(this.$data.constStr.username, user.username);
-          return this.$store.dispatch("userLogin/getUserPermissionsAction");
+          return Promise.all([
+            this.$store.dispatch("userLogin/getUserPermissionsAction"),
+            this.$store.dispatch("userLogin/getUnsupportedFeaturesAction")
+          ]);
         } else {
           throw new Error("Login Failed");
         }
       })
-      .then((res: any) => {
-        if (res) {
+      .then((res: any[]) => {
+        if (res && res.length === 2) {
```

Comment:
why do we want to check for length=2?

Reply:
Don't need to check if length is equal to 2, just length should be enough as response is an array.